helpers/tripModel.js

```diff
@@ -0,0 +1,35 @@
+const db = require('../config/dbConfig');
+const mappers = require('./mappers');
+
+module.exports = {
+  get: function (id) {
+    let query = db('trips');
+
+    if (id) {
+      return query
+        .where('id', id)
+        .first()
+        .then(trip => mappers.tripToBody(trip)); // s
```

Comment:
Since this inner function is just taking in the same argument, you could skip the extra step:
```javascript
.then(mappers.tripToBody);
```